### DISCUSSION TITLE
v0.20.1

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#3157] [Versions & Releases]
## What's Changed
* Drop diffusion from XTTS by @erogol in https://github.com/coqui-ai/TTS/pull/3150
* Bug fixes and add support for multiples speaker references on XTTS inference by @Edresson in https://github.com/coqui-ai/TTS/pull/3149
* Fix XTTS v2.0 training recipe by @Edresson in https://github.com/coqui-ai/TTS/pull/3154

**Full Changelog**: https://github.com/coqui-ai/TTS/compare/v0.20.0...v0.20.1

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.20.1'>v0.20.1</a>.</em>

**Comment by Dantechdevs** [comment]
This is great
